dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -154,7 +154,14 @@ def facilitator_pre_workshop(user, workshop)
   def facilitator_post_workshop(user, workshop)
     @user = user
     @workshop = workshop
-    @survey_url = CDO.studio_url "/pd/misc_survey/facilitator_post", CDO.default_scheme
+    # TODO: After 9/5/2020 move all workshops to the new survey (9/5 is last 2020 Summer Workshop)
```

Comment:
Would you mind making a Jira item for this, if we don't already have one?

Reply:
will do!

Reply:
Jira item is [here](https://codedotorg.atlassian.net/browse/PLC-970) we will want to circle back with MeganH when the time so she knows it's done.